docs/tasks/run-as-deployment/quickdockerbuildusingidentitykeyvault/azuredeploy.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "registryName": {
```

Comment:
where do you use registryName, registrySku and registryAdminUserEnabled in the template?

Reply:
registryName is used in template and I will remove the other two.